compiler/parser/Makefile

```diff
@@ -59,24 +53,14 @@ FORCE:
 include $(COMPILER_ROOT)/make/Makefile.compiler.foot
 
 
-# Use chapel.tab.h as prerequisite, to ensure bison only invoked once, even
+# Use bison-chapel.h as prerequisite, to ensure bison only invoked once, even
 # with parallel execution.
 #
-# no-error required here for bison 1.875 (not 1.875d)
-$(OBJ_SUBDIR)/chapel.tab.o: chapel.tab.h $(OBJ_SUBDIR_MADE) $(LEXYACC_INCLUDES)
-	$(CXX) -c $(COMP_CFLAGS) $(COMP_CFLAGS_NONCHPL) -o $@ chapel.tab.cpp $(OBJ_SUBDIR_MADE) $(LEXYACC_INCLUDES)
-
-
-# no-error required here for flex 2.5.33
-$(OBJ_SUBDIR)/lex.yy.o: lex.yy.cpp $(OBJ_SUBDIR_MADE) chapel.tab.h $(LEXYACC_INCLUDES)
-	$(CXX) -c $(COMP_CFLAGS) $(COMP_CFLAGS_NONCHPL) -o $@ $<
+$(OBJ_SUBDIR)/bison-chapel.o: ../include/bison-chapel.h bison-chapel.cpp $(OBJ_SUBDIR_MADE)
+	$(CXX) -c $(COMP_CFLAGS) $(COMP_CFLAGS_NONCHPL) -o $@ bison-chapel.cpp
 
-#
-# need to handle these ones specially as well on intel because it
-# #includes chapel.tab.h
-#
-$(OBJ_SUBDIR)/parser.o: parser.cpp $(OBJ_SUBDIR_MADE) chapel.tab.h
-	$(CXX) -c $(COMP_CFLAGS) $(COMP_CFLAGS_NONCHPL) -o $@ $<
+$(OBJ_SUBDIR)/flex-chapel.o: ../include/bison-chapel.h ../include/flex-chapel.h flex-chapel.cpp $(OBJ_SUBDIR_MADE)
+	$(CXX) -c $(COMP_CFLAGS) $(COMP_CFLAGS_NONCHPL) -o $@ flex-chapel.cpp
 
-$(OBJ_SUBDIR)/processTokens.o: processTokens.cpp $(OBJ_SUBDIR_MADE) chapel.tab.h
-	$(CXX) -c $(COMP_CFLAGS) $(COMP_CFLAGS_NONCHPL) -o $@ $<
+$(OBJ_SUBDIR)/parser.o:  ../include/bison-chapel.h ../include/flex-chapel.h parser.cpp $(OBJ_SUBDIR_MADE)
+	$(CXX) -c $(COMP_CFLAGS) $(COMP_CFLAGS_NONCHPL) -o $@ parser.cpp
```

Comment:
It looks like we no longer compile processTokens.cpp here.  Is that intentional?  I didn't see the file get removed, is it still used?